tests/shebang/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -6,7 +6,6 @@ exports[`shebang.php 1`] = `
 var_dump($argv);
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #!/usr/bin/php
-<?php
-var_dump($argv);
+<?php var_dump($argv);
```

Comment:
Let's add other test here too, this left as is

Reply:
actually for this one im not sure i agree that prettier should not change the formatting here

Reply:
imo anything inside the tags is php, being formatted by prettier, and is fair game

Reply:
@mgrip agree, just let's add tests:
```php
<?php var_dump($argv);
```

```php
<?php
var_dump($argv);
```